chef/cookbooks/hyperv/recipes/windows_features.rb

```diff
@@ -15,6 +15,7 @@
   node[:features_list][:windows].each do |feature_list|
     windows_feature feature_list do
       action :install
+      all true
```

Comment:
https://github.com/opscode-cookbooks/windows
the all attribute is listed, so it's available